backend/certificates.js

```diff
@@ -34,6 +34,34 @@ export function saveCertsLocally(json) {
     saveSetting('CONNECTOR_HOST_INFO', hostInfo);
     return {};
 }
+
+export function fetchCertsFromCA() {
+    // TODO: What to do when more than one user?
+    // TODO: Should I try getting the api_key if no access_token?
+    const {username, accessToken} = getSetting('USERS')[0];
+    return fetch(
+        `${CA_HOST_URL}`, {
+            method: 'POST',
+            body: JSON.stringify({credentials: {username, access_token: accessToken}})
+        }
+    ).then(res => res.json()).then(json => {
+        return json;
+    });
+}
+
+export function fetchAndSaveCerts() {
+    return fetchCertsFromCA().then(response => {
+        if (!response.error) {
+            Logger.log('Successfully received certs from CA.');
+            if (!saveCertsLocally(response).error) {
+                Logger.log('Saved certificates to local storage.');
+            }
+        } else {
+            Logger.log('An error returned from the CA.' + response.error);
```

Comment:
does the caller of this function need to know if it succeeded or failed? you're returning the promise of all of this (`return fetchCertsFromCA`) which makes it seem like it does, but then not raising any errors or returning any information to indicate success or failure